{
  "totalNumberSkippingTests": 0,
  "bugType": "only_fail",
  "totalNumberErroringTests": 1,
  "repo": "2018swecapstone/h2ms",
  "failingModule": "/root/workspace/2018swecapstone/h2ms/371835929",
  "hostname": "spirals-vortex.lille.inria.fr",
  "totalNumberFailingTests": 0,
  "error-types": [
    "org.springframework.web.util.NestedServletException"
  ],
  "totalNumberRunningTests": 114,
  "failing-test-cases": [
    {
      "className": "edu.harvard.h2ms.controllers.UserControllerTests",
      "failingMethods": [],
      "erroringMethods": [
        "edu.harvard.h2ms.controllers.UserControllerTests#test_AdminRequest_UserController_saveNewUser"
      ],
      "failures": [
        {
          "failureName": "org.springframework.web.util.NestedServletException",
          "failureDetail": "Request processing failed; nested exception is javax.validation.ConstraintViolationException: Validation failed for classes [edu.harvard.h2ms.domain.core.User] during persist time for groups [javax.validation.groups.Default, ]\nList of constraint violations:[\n ConstraintViolationImpl{interpolatedMessage\u003d\u0027may not be null\u0027, propertyPath\u003dpassword, rootBeanClass\u003dclass edu.harvard.h2ms.domain.core.User, messageTemplate\u003d\u0027{javax.validation.constraints.NotNull.message}\u0027}\n]",
          "isError": true
        }
      ],
      "nbFailures": 0,
      "nbErrors": 1
    }
  ]
}